### PR TITLE
resolve moment path when addon is being built by ember build

### PR DESCRIPTION
**Pull request by zomarg**

### CODE REVIEW

**Comment by zomarg** [comment]
#18 
@knownasilya  tested:
npm pack (on addon dir)
created new ember app and did
npm install ../toLocalFileSystem.tgz
ember g ember-toastr

Everything went fine.

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "ember-toastr",
-  "version": "1.6.0",
+  "version": "1.6.1",
```
Comment (knownasilya) [inline package.json:3]: Could you remove the bump. I bump specifically after I test locally.

Comment (zomarg) [reply package.json:3]: No dev computer now but if you can wait till evening i can do it.

Comment (zomarg) [reply package.json:3]: @knownasilya  beside that you said to put brocoli-funnel and broccoli-merge-trees as devDependencies I am still for putting them as dependecies. Its your call.

Comment (knownasilya) [reply package.json:3]: I'll have to fix that in the next release, since I already published this fix.

**Pull request merged**